ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/useNodeVulnerabilities.ts

```diff
@@ -19,8 +20,7 @@ const nodeVulnerabilitiesQuery = gql`
 export default function useNodeVulnerabilities(
     id: string,
     query: string,
-    page: number,
-    perPage: number
+    { page, perPage, sortOption }: ClientPagination
```

Comment:
Following the style of the API change in the PR before this one, moving these values to an object instead of positional params.